src/modal.coffee

```diff
@@ -96,6 +99,10 @@ Ember.Component.extend Ember.Widgets.StyleBindingsMixin,
 
   didInsertElement: ->
     @_super()
+    # Make sure that after the modal is rendered, set focus to the first
+    # tabbable element
+    Ember.run.schedule 'afterRender', this, ->
```

Comment:
You will need to cancel it in case someone destroys the modal right after inserting it in the DOM